skyve-war/src/main/webapp/external/selfRegister.xhtml

```diff
@@ -26,8 +26,9 @@
 			<ui:param name="showLogout" value="false" />
 			<ui:define name="head">
 				<!-- Password strength estimator -->
-				<script src="https://cdnjs.cloudflare.com/ajax/libs/zxcvbn/4.2.0/zxcvbn.js"></script>
-                <link rel="stylesheet" type="text/css" href="pages/css/strengthIndicator.css" />
+				<script type="text/javascript" src="strength-indicator/zxcvbn.js"></script>
+				<script type="text/javascript" src="strength-indicator/util.js"></script>
+				<link rel="stylesheet" type="text/css" href="pages/css/strength-indicator.css"></link>
```

Comment:
Can this file be moved into the skyve-web jar and in the same folder as the zxcvbn and util js? Then it doesn't need to be part of the admin module.